docs/source/tutorials/Visualisation_assembly_reads_pileup_IGV.md

```diff
@@ -0,0 +1,96 @@
+# Visualizing assembly and associated reads pileup with IGV
+
+ The [IGV: Integrative Genomics Viewer](http://software.broadinstitute.org/software/igv/home) software allows to visulally explore and scroll through sequences, that may or not be splitted in contigs, and may or not be associated with reads pileup. It can eg. allow you to detect specific positions were variants are errorneous (eg. at the end of contigs where reads are poorly mapped) and visualize eg. locations where coverage is not homogeneous (ie. repeated regions that are not proprely assembled).
+
+ NB: Convention: means `<change_me>`
+
+## 1. Getting IGV
+
+- you can install IGV on your pc do so.
+- you can use SAGA with a graphical user interface
+
+### 1.1. Installing IGV on your PC
+If want to do that at your PC you can install IGV as such
```

Comment:
Replace with link to online install

Reply:
We had spoken of (had added)
- online - + security
- local installation + added software link from VI as George did request
- virtual box install in linux OR linux + mac